specification/ai/Azure.AI.Client/common.tsp

```diff
@@ -0,0 +1,30 @@
+import "@typespec/rest";
```

Comment:
Why is that in common.tsp? should this not be in evaluations\models.tsp?

Reply:
This is definition for SystemData -- which included info about who created the entity etc. So that can be common hence put it here.